### ISSUE TITLE
Native image no longer builds

### ISSUE THREAD

**Issue by tomas-langer** [oracle/helidon-build-tools#285]
The newest version (2.1.0) now fails to build native image with the following exception:
```
[INFO] Building native image :/Users/tomas/dev/helidon/helidon/tests/integration/native-image/mp-1/target/helidon-tests-native-image-mp-1
[WARNING] Error: Invalid Path entry "/Users/tomas/dev/helidon/helidon/tests/integration/native-image/mp-1/target"
[WARNING] Caused by: java.nio.file.NoSuchFileException: /Users/tomas/dev/helidon/helidon/tests/integration/native-image/mp-1/"/Users/tomas/dev/helidon/helidon/tests/integration/native-image/mp-1/target"
```

When I downgraded to `2.0.2`, everything works fine.

**Comment by tomas-langer** [comment]
Fixed in 2.1.1, closing again

**Comment by romain-grecourt** [comment]
See https://github.com/oracle/helidon-build-tools/pull/283
